packages/paste-website/src/pages/components/input/index.mdx

```diff
@@ -301,7 +300,7 @@ Use a read-only form field when a field's value can't be changed, but users shou
 
 ## Composition notes
 
```

Comment:
suggestion: imo we could just remove this whole first section "An input field must have..." + its example.